pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/RetentionPolicies.java

```diff
@@ -28,24 +28,33 @@
  * Infinite retention can be achieved by setting both time and size limits to `-1`.
  */
 public class RetentionPolicies {
-    private int retentionTimeInMinutes;
-    private long retentionSizeInMB;
+
+    public static final int DEFAULT_RETENTION_TIME_IN_MINUTES = 0;
+
+    public static final long DEFAULT_RETENTION_SIZE_IN_MB = 0;
+
+    private Integer retentionTimeInMinutes;
+    private Long retentionSizeInMB;
```

Comment:
This will introduce a compatibility issue when downgrading to an old version that does not allows 
 `retentionTimeInMinutes` and `retentionSizeInMB` to zero?

Reply:
There are no compatibility issues and it can be set to 0. But this PR does not solve the problem that only one property is set correctly, so I will close this PR.